tethysext/atcore/services/workflow_manager/condor_workflow_manager.py

```diff
@@ -200,7 +200,13 @@ def prepare(self):
         # Parametrize each job
         for job in self.jobs:
             # Set arguments for each job
-            job.set_attribute('arguments', self.job_args)
+            existing_job_args = job.get_attribute('arguments')
+            if existing_job_args and type(existing_job_args) != list:
+                job_args_list = []
+                job_args_list.append(existing_job_args)
+                existing_job_args = job_args_list
```

Comment:
If this is retained, it can be simplified to `existing_job_args = [exiting_job_args]`.

Reply:
Thanks.